src/components/MapBox.tsx

```diff
@@ -286,7 +286,7 @@ export const MapBox: React.FC = () => {
                     <Popup>
                       <SearchCard
                         key={f.id}
-                        sm={true}
+                        mapCard={true}
```

Comment:
if i read `mapCard` i don't know what it happening in the searchCard. Is it showing a mapCard, or?

Can you make the naming more descriptive?

Reply:
card on the map

Reply:
so this shows it? and if it is `fals`e it doesn't show the map on the card?

If so, can you make it descriptive in the naming that is shown or not shown, e.g. `showMapCard` or `isMapCardShown` ?

Reply:
`MapCard` and `Results` are still in the components folder, but should be moved to the container folder.